src/System.Private.CoreLib/src/System/Activator.cs

```diff
@@ -164,7 +164,7 @@ public static ObjectHandle CreateInstance(string assemblyName, string typeName,
                                                 culture,
                                                 activationAttributes);
 
-            return (o != null) ? new ObjectHandle(o) : null;          
```

Comment:
You should be able to hit this path with null for `Nullable<int>`

Reply:
Thank's for tip!